adventure.go

```diff
@@ -75,11 +74,24 @@ func loadRooms() {
 	//		fmt.Println(j)
 	//	}
 	//}
+
+	// Debug uncomment to print room exits
+	/*
+		for _, i := range rooms {
+			fmt.Printf("%s:\n", i.Name)
+			for _, j := range i.Exits {
+				fmt.Printf("\t%s\n", j)
+			}
+		}
+	*/
 }
 
 func main() {
 	loadRooms()
 
+	// TODO start room must be initialized with Visited = True
+	curRoom = rooms[0]
```

Comment:
We'll need to make sure we're always starting in the attic.  Should we just hard-code that here, or do you have other ideas about how to do that?

Reply:
The start room will need to be initialized here. It will need to hard coded and marked as visited.